# HG changeset patch
# User Rik <rik@octave.org>
# Date 1363025629 25200
#      Mon Mar 11 11:13:49 2013 -0700
# Node ID 6601c66280c45ba1f7099206267c116d1842d1cb
# Parent  4dc4da5970e53cb1a508fb4ad1b21c28177a5780
doc: Mention octave_config_info ("features") as new way of looking for HAVE_XXX targets.

* doc/interpreter/testfun.txi: Mention octave_config_info ("features") as new
way of looking for HAVE_XXX targets.

diff --git a/doc/interpreter/testfun.txi b/doc/interpreter/testfun.txi
--- a/doc/interpreter/testfun.txi
+++ b/doc/interpreter/testfun.txi
@@ -136,17 +136,17 @@ block will only be run if Octave was com
 @code{svds()}, depends on having the @sc{arpack} library.  All of the tests
 for @code{svds} begin with
 
 @example
 %!testif HAVE_ARPACK
 @end example
 
 @noindent
-Review @file{config.h} or @code{octave_config_info ("DEFS")} to see some
+Review @file{config.h} or @code{octave_config_info ("features")} to see some
 of the possible values to check.
 
 Sometimes during development there is a test that should work but is
 known to fail.  You still want to leave the test in because when the
 final code is ready the test should pass, but you may not be able to
 fix it immediately.  To avoid unnecessary bug reports for these known
 failures, mark the block with @code{xtest} rather than @code{test}:
 
